### ISSUE TITLE
Question on how to interpret the data for typescript

### ISSUE THREAD

**Issue by avkonst** [compat-table/compat-table#1088]
I am browsing the great table here http://kangax.github.io/compat-table/es6/
I am writing my code in typescript and it's compiler can target es5 and es6 with various other different options. Let's say I pick es6 target, what column should I browse for compatibility support? Still typescript+corejs or actual columns for browsers? I understand columns for browsers... but what does column for typescript+corejs really shows, how does it incorporate variation in compiler's target and options?

**Comment by afmenez** [comment]
I agree, the transpiler columns are not very clear currently.  The doc says the output must be tested with an old browser with ES3 support, so I assume transpilers which cannot target ES3 should not be listed.
I believe a better approach would be to have multiple columns when the transpiler has multiple targets (TS->ES5, TS->ES6, etc).
